Makefile

```
@@ -469,6 +499,33 @@ stt-golden:
 	TLDW_STT_GOLDEN_AUDIO_DIR="$(STT_GOLDEN_AUDIO_DIR)" \
 	python -m pytest tldw_Server_API/tests/Audio/test_stt_adapters_golden.py -m "stt_golden" -v
 
+# -----------------------------------------------------------------------------
+# Sanity check (C2)
+# -----------------------------------------------------------------------------
+.PHONY: check
+
+check:
```

checkmake (0.2.2)

[warning] Target body for "check" exceeds allowed length of 5 (6).

(maxbodylength)